tests/test__greenness.py

```diff
@@ -4,8 +4,15 @@
 """
 import unittest
 
-from eventlet.green import urllib2, BaseHTTPServer
+from eventlet.green import BaseHTTPServer
 from eventlet import spawn, kill
+from eventlet.support import six
+
+if six.PY2:
+    from eventlet.green.urllib2 import HTTPError, urlopen
```

Comment:
`from ...green import urllib2`
importing module level names tends to bring more harm than convenience later on.